discovery.md

```diff
@@ -22,10 +22,13 @@ version.
 ## Overview
 
 The goal of the CloudDiscovery API is to enable connections between consumers of
-events and Event providers / producers / aggregators / brokers of events and to
-facilitate the creation of CloudSubscriptions.
+events and Event producers to facilitate the creation of
```

Comment:
+ a

_".. facilitate the creation of a CloudEventSubscription. .."_

Reply:
Done